cmd/serve/main.go

```diff
@@ -215,25 +215,12 @@ func (cmd *servecmd) Run() {
 	router.PathPrefix("/health").Handler(services.HealthEndpoint())
 
 	// Authentication middleware
-	if conf.Auth.IsEnabled() {
-		middleware, err := authnz.NewMiddleware(conf.Auth, claimsMiddleware(conf))
-		if err != nil {
-			logrus.Error(errors.Wrap(err, "failed to set up authnz middleware"))
-			return
-		}
-		router.Use(middleware)
-	} else {
-		logrus.Warn("[DEPRECATION NOTICE] using wg-access-server without an admin user is deprecated and will be removed in an upcoming minor release.")
```

Comment:
just FYI, this could never be triggered since admin password was required, and admin password enabled basic auth so I just removed it to make it clearer